server/src/test/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryApplicationTest.java

```diff
@@ -0,0 +1,106 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.opentosca.toscana.core.BaseUnitTest;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class CloudFoundryApplicationTest extends BaseUnitTest {
+
+    private final static CloudFoundryApplication myApp = new CloudFoundryApplication();
+    private String testService1;
+    private String testService2;
+    private String appName;
+    private String pathToTest;
+    private String test2;
+    private String testValue2;
+    private String testAttribut2;
+    private String attributValue1;
+    private String attributValue2;
+    private String buildPack;
+
+    @Before
+    public void setUp() {
+        String test1 = "TEST1";
+        String test3 = "TEST3";
+        String testAttribut1 = "TEST_ATTRIBUTE1";
+        testService1 = "TEST_SERVICE1";
+        testService2 = "TEST_SERVICE2";
+        pathToTest = "/path/to/test";
+        appName = "testApp";
+        test2 = "TEST2";
+        testValue2 = "TEST_VALUE2";
+        attributValue1 = "ATTRIBUT1";
+        testAttribut2 = "TEST_ATTRIBUTE2";
+        attributValue2 = "ATTRIBUT2";
+        buildPack = "TEST_BUILDPACK";
+
+        myApp.setAppName(appName);
+
+        myApp.addEnvironmentVariables(test1);
+        myApp.addEnvironmentVariables(test2, testValue2);
+        myApp.addEnvironmentVariables(test3);
+
+        myApp.addService(testService1);
+        myApp.addService(testService2);
+
+        myApp.addFilePath(pathToTest);
+
+        myApp.addAttribute(testAttribut1, attributValue1);
+        myApp.addAttribute(testAttribut2, attributValue2);
+        
+        myApp.addBuildpack(buildPack);
+    }
+
+    @Test
+    public void getAppName() {
+        assertEquals(appName, myApp.getAppName());
+    }
+
+    @Test
+    public void getEnvironmentVariables() {
+        Map<String, String> env = myApp.getEnvironmentVariables();
+
+        assertTrue(env.size() == 3);
```

Comment:
Always try to use more specialized assert methods. here:
`assertEquals(3, env.size());`
It really matters, because in case of test failure, the test log will contain smth like:
`was supposed to be 3, but is <actual value>`